Docs: Fix Starting Container Instructions for RocksDB Iroha (#1600)
hyperledger/iroha: docs/source/getting_started/index.rst

Creating a Docker Network
^^^^^^^^^^^^^^^^^^^^^^^^^

> To operate, Iroha requires a ``PostgreSQL`` database, if you use Iroha with PostgreSQL and not RocksDB.

Reviewer: ```suggestion
To operate, Iroha requires a ``PostgreSQL`` database or ``RocksDB``.
```

Author: @kuvaldini  This is not working well within the context. But I added some more and it should be ok now, so please take a look

  -e KEY='node0' \
  hyperledger/iroha:latest

> .. note:: If you are using Iroha without PostgreSQL (with RocksDB), please add ``--entrypoint irohad`` to these parameters, to allow Iroha to skip the PostgreSQL container check.

Reviewer: ```suggestion
.. note:: If you are using Iroha with RocksDB, please add ``--entrypoint irohad`` to these parameters, to skip the PostgreSQL container check.
```
